metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/enrichment_commands.py

```diff
@@ -232,3 +241,48 @@ def set_hbase_acls(self):
 
         Logger.info("Done setting HBase ACLs")
         self.set_hbase_acl_configured()
+
+    def service_check(self, env):
+        """
+        Performs a service check for Enrichment.
+        :param env: Environment
+        """
+        Logger.info("Checking for Geo database")
+        metron_service.check_hdfs_file_exists(self.__params, self.__params.geoip_hdfs_dir + "/GeoLite2-City.mmdb.gz")
```

Comment:
You know, honestly, the better approach here, unfortunately, is to pull the config from the global config and ensure that the file exists.  What if the user renames the GeoLite database to something other than `GeoLite2-City.mmdb.gz`?  Can we make that a follow-on JIRA at least?

Reply:
I agree.  I don't like hard-coding "GeoLite2-City.mmdb.gz" here.  I just didn't see anywhere in the Mpack where we had that value parameterized already.

Where in the code do we have it parameterized?  I'd love to fix this.

Reply:
Ok, I see it now under a global properties key "geo.hdfs.file".  There is nothing in Ambari MPack for it, which might complicate using it.  I am just thinking through, if we would need to first introduce it as an Ambari-managed configuration value.

Reply:
Yeah, we just have the HDFS directory, which is what you're using.  For this PR, I think it's good.  I'd hate to make the perfect the enemy of the good.

Going forward, what would be cool is if we could execute a stellar script via the REPL and if it fails, fail the service check.  Since the REPL can interact with global config parameters and has the capability to validate HDFS, that would be a clean way to do this.